Automattic/jetpack
# Add video preview to Social media picker

Add a video preview to the media picker.

## Proposed changes:

* Show a video preview when user picks a video file.
* Play the video file when a user hovers the video.
* Show a timer that counts down while playing.

### Other information:


## Jetpack product discussion


p1HpG7-jqQ-p2

## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






- Add support for enhanced publishing to your site (context here: D90213-code)
- Start a new post. 
- Check that you can pick a video. (see https://sample-videos.com/ for videos to use)
- Check that you can replace the video by clicking on the current one.
- Check that you can remove the video.
- Check that the video starts playing and the timer starts counting down on hover.
- Make sure the tests pass: cd projects/js-packages/publicize-components && npm test src/components/media-section/test/index.test.js

<img width="266" alt="image" src="https://user-images.githubusercontent.com/1713699/215120107-4f032530-8f97-4bbb-b7ae-c09051808aa5.png">
